components/error-summary/src/__snapshots__/test.js.snap

```diff
@@ -93,6 +93,7 @@ exports[`error summary matches the ErrorSummary snapshot: error summary 1`] = `
                 escapeHtml={false}
                 linkRenderer={[Function]}
                 mb={3}
+                parserOptions={Object {}}
```

Comment:
odd, any idea what this is?

Reply:
nope - snapshots be crazy bro!

(it's probably the result of a minor bump in *one* of the modules updated in this PR)

Reply:
maybe react markdown got an upgrade and this is a default prop